client/tests/test-1.spec.ts

```diff
@@ -0,0 +1,44 @@
+import { test, expect } from '@playwright/test';
+
+test.beforeEach(async ({ page }) => {
+  await page.goto('http://localhost:5173');
+});
+test('test login send message and logout', async ({ page }) => {
+  await page.getByPlaceholder('Email').click();
+  await page.getByPlaceholder('Email').fill('ofek@gmail.com');
+  await page.getByPlaceholder('Email').press('Tab');
+  await page.getByPlaceholder('Password').fill('O!f12y345678');
+  await page.getByPlaceholder('Password').press('Enter');
+  await expect(page.getByText('Select chat...')).toBeVisible();
+  await page.locator('p').filter({ hasText: 'avi' }).click();
```

Comment:
I'm flattered.
But here as well, don't base your tests on the existing state of the app.
At a later point, we want to test clean — the CI will run the app, and run the tests. It can't be dependent on a specific state.
We want it also to be automatic locally, so you don't need to run and test, but create a script that runs and tests.
It can be done easily with NPM.
Check if there's a way to mock the db.

Reply:
See https://github.com/valkey-io/valkey-glide/blob/main/node/tests/AsyncClient.test.ts#L15 for example, we're creating a new DB and running tests on it, no data is assumed.